src/interfaces/resource-interfaces/resource-interface.ts

```diff
@@ -18,7 +18,8 @@ export interface ResourceIF {
       subtitle: string
       helpSection?: HelpSectionIF
     }
-    nameRequestTypes?: Array<NameRequestTypes>
+    nameRequestTypes?: Array<NameRequestTypes>,
```

Comment:
We don't need commas between properties in an interface.

I don't care which way we go here as long as it's consistent.